### DISCUSSION TITLE
Can I control audio forwarding using Test Distribution (this dependency) ?

### DISCUSSION THREAD

**Discussion by EvilSprit** [AppiumTestDistribution/AppiumTestDistribution#1184] [Q&A]
The requirement with my product is I require to test NLP using dummy audio file which requires audio playback support. A

Currently I'm running the automation using Virtual cable in serial fashion. As my product is growing,  it is getting difficult to run test cases in serial fashion.

Can you please confirm or provide support for our problem. 

@saikrishna321

**Comment by saikrishna321** [comment]
As ur already ur test in serial, ATD can be used to parallelise ur test.
